lib/versioner.js

```diff
@@ -27,6 +32,30 @@ function versionValidish(version) {
   return [major, minor, patch].join('.');
 }
 
+/**
+ * Helper method to sort pairs of [versionNumber, function]
+ *
+ * @param {Array} pair [version, handler]
+ *   Pair of [version, handler] to be tested
+ *
+ * @param {Array} pair [version, handler]
+ *   Pair of [version, handler] to be tested
+ *
+ * @return {f}
+ *   A sorter function that will return 1 if first argument version is greater
+ *   than the second.
+ */
+const versionSorter = sort(([a], [b]) => {
+  if (a === '*') {
+    return 1;
+  }
+  else if (b === '*') {
+    return -1;
+  }
+
+  return a < b;
```

Comment:
I think this comparison is insufficient since we're dealing with version _strings_, e.g.

```
'v3.10.0' < 'v3.2.0' // true
```

`semver` does have a `compare` method which should sort this out though.

Reply:
Oh, you're completely right, sorry for that.